src/libpmemset/pmemset.c

```diff
@@ -363,6 +392,8 @@ pmemset_first_part_map(struct pmemset *set, struct pmemset_part_map **pmap)
 
 	if (first)
 		*pmap = ravl_interval_data(first);
+
+	(*pmap)->refcount += 1;
```

Comment:
this should be incremeneted in the part.c file. Create a function that does that. Same below.